app/assets/javascripts/govuk_publishing_components/analytics-ga4/gtm-page-views.js

```diff
@@ -28,6 +28,15 @@
             level_1: this.getMetaContent('themes'),
             topics: this.getMetaContent('taxon-slugs'),
             topics_ids: this.getMetaContent('taxon-ids')
+          },
+          content: {
+            language: this.getLanguage(),
+            history: this.getHistory(),
+            withdrawn: this.getWithDrawn(),
+            government: this.getMetaContent('government'),
+            political_status: this.getMetaContent('political-status'),
+            org_ids: this.getMetaContent('analytics:organisations'),
+            world_location: this.getMetaContent('analytics:world-locations')
           }
```

Comment:
Quite an intense amount of data - assuming it's all used I can see why the data teams are keen to get this in - it seems it pretty much covers everything!

Reply:
Yes, there's a lot - and I haven't finished implementing it yet either - see https://docs.google.com/document/d/1LGLRb6zMvJBjzMYPgLAcBWbS-GsKGUzLcumCsZgiUhM/edit#